sdk/spring/azure-spring-boot-2-2-starter-cosmosdb/README.md

```diff
@@ -0,0 +1,52 @@
+# Azure Cosmos DB Spring Boot 2.2 Starter client library for Java
+
+[Azure Cosmos DB](https://azure.microsoft.com/services/cosmos-db/) is a globally-distributed database service that allows developers to work with data using a variety of standard APIs, such as SQL, MongoDB, Graph, and Azure Table storage. 
+
+[Package (Maven)][package] | [API reference documentation][refdocs] | [Product documentation][docs] | [Samples][sample]
+
+## Getting started
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Maven](http://maven.apache.org/) 3.0 and above
+
+### Include the package
+[//]: # ({x-version-update-start;com.microsoft.azure:azure-cosmosdb-spring-boot-2-2-starter;current})
+```xml
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-cosmosdb-spring-boot-2-2-starter</artifactId>
+    <version>2.4.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+## Key concepts
+Please refer to [Azure Cosmos DB Spring Boot 2.3 Starter](../azure-spring-boot-2-3-starter-cosmosdb/README.md) for key concepts.
```

Comment:
Let's not use relative links here. https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-TEMPLATE.md#link-guidelines

And we should link to the `Key concepts` part of the `azure-spring-boot-2-3-starter-cosmosdb/README.md`.